common/package.json

```diff
@@ -1,13 +1,35 @@
 {
   "name": "openpassport-common",
   "version": "0.0.1",
+  "description": "interal constants and utils",
   "license": "MIT",
   "author": "",
+  "private": true,
+  "type": "module",
+  "main": "dist/index.js",
+  "exports": {
+    ".": "./dist/index.js",
+    "./constants/*": "./dist/constants/*.js",
+    "./utils/*": "./dist/utils/*.js",
+    "./ofacdata/outputs/nameAndDobSMT.json": "./ofacdata/outputs/nameAndDobSMT.json",
+    "./ofacdata/outputs/nameAndYobSMT.json": "./ofacdata/outputs/nameAndYobSMT.json",
+    "./ofacdata/outputs/passportNoAndNationalitySMT.json": "./ofacdata/outputs/passportNoAndNationalitySMT.json",
+    "./pubkeys/serialized_dsc_tree.json": "./pubkeys/serialized_dsc_tree.json",
+    "./mock_certificates/*": "./src/mock_certificates/*",
+    "./mock_certificates/**/*": "./src/mock_certificates/**/*"
+  },
+  "files": [
+    "dist",
+    "src/mock_certificates"
+  ],
   "scripts": {
+    "build": "tsc",
+    "types": "yarn build",
```

Comment:
this is here so we can run `yarn types` from root and know that in ever package it is compilin the typescript files since build is sometimes used for that and sometimes not